### PR TITLE
TW-991: removing new group chat in new dm

### PR DESCRIPTION
**Pull request by imGok**
# DOD: 

- [x] Encrypted Chat
- [x] Unencrypted Chat
- [x] Group Creation

# WEB DEMO

## Encrypted Chat

https://github.com/linagora/twake-on-matrix/assets/48354990/5872883b-f293-40dd-b9c4-b50619788e55

## Unencrypted

https://github.com/linagora/twake-on-matrix/assets/48354990/69b2f69a-6f25-4211-8875-f0bb3a0d76fa

## Group Chat

https://github.com/linagora/twake-on-matrix/assets/48354990/599ec5b0-58a8-4c93-a38e-f226e06faa5e

# MOBILE DEMO 

## Encrypted

https://github.com/linagora/twake-on-matrix/assets/48354990/c19a24f0-a260-4188-aee7-7749983d4812

## Unencrypted 

https://github.com/linagora/twake-on-matrix/assets/48354990/76ab8135-7ee2-4407-88d0-43c5f2832f8b

## Group Chat

https://github.com/linagora/twake-on-matrix/assets/48354990/7c585f1e-8015-4ad0-a6d0-6925124cb2ad

#991

### CODE REVIEW

**Comment by hoangdat** [comment]
Hi @imGok , In mobile, no way to `Create Group Chat`

**Comment by hoangdat** [comment]
Hi @imGok can we create unencrypted chat and encrypted chat with the same people?

**Comment by imGok** [comment]
> Hi @imGok can we create unencrypted chat and encrypted chat with the same people?

clicking on a person that you already chatted with will just open the room and show messages

File: lib/pages/chat/chat_app_bar_title_style.dart [hunk 1]
```diff
@@ -6,8 +6,10 @@ import 'package:linagora_design_flutter/linagora_design_flutter.dart';
 class ChatAppBarTitleStyle {
   static ResponsiveUtils responsive = getIt.get<ResponsiveUtils>();
 
-  static Color get currentlyActiveColor => const Color(0xFF5AD439);
-  static Color get currentlyInactiveColor => const Color(0xFF818C99);
+  static const Color currentlyActiveColor = Color(0xFF5AD439);
+  static const Color currentlyInactiveColor = Color(0xFF818C99);
+  static const Color encryptedIconColor = Color(0xFF00C853);
```
Comment (sherlockvn) [inline lib/pages/chat/chat_app_bar_title_style.dart:11]: please check the design to see if the color is updated or not

File: lib/config/first_column_inner_routes.dart [hunk 2]
```diff
@@ -33,8 +35,13 @@ class FirstColumnInnerRoutes {
           const NewGroup(),
         );
       case 'innernavigator/newprivatechat':
+        final argumentsCast =
+            tryCast<Map<String, bool>>(arguments, fallback: null);
         return _defaultPageRoute(
-          const NewPrivateChat(),
+          NewPrivateChat(
+            enableEncryption:
+                argumentsCast?[DraftChatConstant.enableEncryption] ?? true,
```
Comment (hoangdat) [inline lib/config/first_column_inner_routes.dart:43]: why `true`? must disable encrypted by default

File: lib/presentation/mixins/go_to_direct_chat_mixin.dart [hunk 3]
```diff
@@ -32,6 +37,7 @@ mixin GoToDraftChatMixin {
     required BuildContext context,
     required String path,
     required ContactPresentationSearch contactPresentationSearch,
+    bool enabledEncryption = true,
```
Comment (hoangdat) [inline lib/presentation/mixins/go_to_direct_chat_mixin.dart:40]: false by default.

File: lib/presentation/mixins/go_to_direct_chat_mixin.dart [hunk 4]
```diff
@@ -103,4 +114,41 @@ mixin GoToDraftChatMixin {
     if (roomIdResult.error != null) return;
     context.go('/$path/${roomIdResult.result!}');
   }
+
+  void goToNewPrivateChat(
+    BuildContext context, {
+    bool enableEncryption = true,
```
Comment (hoangdat) [inline lib/presentation/mixins/go_to_direct_chat_mixin.dart:120]: false by default

File: lib/presentation/model/draft_chat_constant.dart [hunk 5]
```diff
@@ -0,0 +1,3 @@
+class DraftChatConstant {
```
Comment (hoangdat) [inline lib/presentation/model/draft_chat_constant.dart:1]: `DraftChatArgument`

Comment (hoangdat) [inline lib/presentation/model/draft_chat_constant.dart:1]:
```suggestion
class DraftChatArgument {
   .....
}
```

**Pull request closed without merging**